### DISCUSSION TITLE
Welcome to tinysleepnet Discussions!

### DISCUSSION THREAD

**Discussion by akaraspt** [akaraspt/tinysleepnet#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ssggf** [comment]
hello! Can you share the tinysleepnet-best_ckpt model file？plz

**Comment by MahdadJafarzadeh** [comment]
(1) Is it feasible to use this CNN-LSTM stack in real-time?
(2) I try to make only the CNN part (skipping the LSTM appending to it), but the use_rnn = False inside train.py gives an error. How can I make only the CNN part?

**Reply by ssggf** [reply]
I have trained the model in whole network. It seem well in test data, however bad in other data in the sleep-edf database which not used in train and test dataset. So fi use this in real-time, i don't think it work.

**Comment by BruceYang521** [comment]
to run at step. 2
python prepare_sleepedf.py

signals = psg_f.readSignal(select_ch_idx).reshape(-1, n_epoch_samples)
ValueError: cannot reshape array of size 7950000 into shape (90000)
anyone can help me?

**Comment by Decoder11** [comment]
ImportError: cannot import name 'pretrain' from 'config' (/Users/atharvnadkarni/opt/anaconda3/lib/python3.9/site-packages/config/__init__.py)

This is the error that it gives me trying to copy the code. What should I do. Please help
